### PR TITLE
SO-4946 Remove old mrcm model

### PR DESCRIPTION
**Pull request by AAAlinaaa**

### CODE REVIEW

File: snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/io/SnomedRefSetDSVExportTest.java [hunk 1]
```diff
@@ -156,17 +156,7 @@ private String createBranch(String branchName) {
...
-
-	private Iterable<SnomedConstraint> getConstraints(String branchPath, SnomedConcepts concepts) {
-		return SnomedRequests
-				.prepareGetApplicablePredicates(bus, branchPath, 
-						idsOf(concepts), 
-						ancestorsOf(concepts), 
-						Collections.emptySet(), 
-						relationshipKeysOf(concepts))
-				.getSync();
+		return transformToExportItems();
```
Comment (cmark) [inline snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/io/SnomedRefSetDSVExportTest.java]: Do these constraints play a role in these tests? If yes, please keep them commented out and add a NOTE about why they are commented out.

**Review by cmark — CHANGES_REQUESTED** [review]

**Review by apeteri — APPROVED** [review]
LGTM once remaining concerns are addressed.

**Review by cmark — APPROVED** [review]
LGTM!

**Review by nagyo — APPROVED** [review]
LGTM!

**Pull request merged**